health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/bill_inbox/BillInbox.js

```
@@ -0,0 +1,418 @@
+            <div style={{ display: "flex", flexDirection: "row", gap: "24px", marginBottom: showGenerateBillAction && BillData?.bills?.length === 0 && !isBillLoading && !isFetchingBill && billGenerationStatus == null ? "2.5rem" : "0px" }}>
+                <div style={{ width: "30%", display: "flex", flexDirection: "column", gap: "24px" }}>
+                    <div style={{ display: "flex", flexDirection: "row" }}>
+                        <InboxSearchLinkHeader
+                            headerText={"HCM_AM_BILL_INBOX"}
+                            links={[
+                                {
+                                    url: "/employee/payments/my-bills",
+                                    text: "HCM_AM_MY_BILLS",
+                                },
+                            ]}
+                        ></InboxSearchLinkHeader>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)